llvm/lib/Target/DirectX/DXILOpLowering.cpp

```diff
@@ -72,10 +75,39 @@ static SmallVector<Value *> argVectorFlatten(CallInst *Orig,
   return NewOperands;
 }
 
+static VersionTuple getShaderModelVer(Module &M) {
```

Comment:
Both getShaderModelVer and getShaderKind will be used in many places, could we have them shared instead of a static function in DXILOpLowering.cpp?

Reply:
> Both getShaderModelVer and getShaderKind will be used in many places, could we have them shared instead of a static function in DXILOpLowering.cpp?

Deleted the functions as each one is called once and in-lined the functionality.